source/features/quick-comment-edit.tsx

```diff
@@ -59,5 +60,7 @@ void features.add(import.meta.url, {
 	include: [
 		pageDetect.hasComments,
 	],
+	// The feature is "disabled" via CSS selector when the conversation is locked.
+	// We want the edit buttons to appear while the conversation is loading, but we only know it's locked when the page has finished.
```

Comment:
I did not see any changes in the comment’s DOM, they're all marked as "editable".

If anyone sees any differences, please leave a comment or send a PR to move this check from CSS to the `addQuickEditButton` function (which should be able to tell whether the current comment is editable)

It would be even greater if this could be merged into a single "can comment be edited by current user", replacing the whole selector-based logic we have

Reply:
WIP: It might be possible via `.comment .js-update-minimized-content` 🤔